jobs/JGLOBAL_WAVE_POST_SBS

```diff
@@ -19,7 +19,18 @@ for out_dir in "${COMOUT_WAVE_PREP}" "${COMOUT_WAVE_GRID}"; do
     if [[ ! -d "${out_dir}" ]]; then mkdir -p "${out_dir}"; fi
 done
 
-
+if [[ -n "${wavepostGRD}" || -n "${waveinterpGRD}" ]]; then
+    for grdID in ${wavepostGRD} ${waveinterpGRD}; do
+        source "${USHgfs}/wave_domain_grid.sh" 
+        process_grdID "${grdID}"
+        YMD=${PDY} HH=${cyc} GRDRESNAME=${grdNAME} declare_from_tmpl -rx \
+            "COMOUT_WAVE_GRID_${GRDNAME}_${GRDRES}:COM_WAVE_GRID_RES_TMPL"
+        out_dir=${COMOUT_WAVE_GRID}_${GRDNAME}_${GRDRES}
+        if [[ ! -d "${!out_dir}" ]]; then mkdir -p "${!out_dir}"; fi
```

Comment:
For my knowledge moving forwad, I've seen a lot of `${!out_dir}`  when I would typically just except to see ${out_dir}, can you help me understand why for the future.

Reply:
`!` is the indirection operator for bash. It says expand on the value of the variable instead of the variable:
```
a=b
b=c
echo ${a} # prints b
echo ${!a} # prints c (${!a} gets evaluated to ${b}, which then get evaluated to c)
```
That said, I've already recommend reducing the number of indirections.

Reply:
Thank you for taking time to explain this to me @WalterKolczynski-NOAA! It is really appreciated.